### PR TITLE
Cm/pse 6 rework

### PR DESCRIPTION
**Pull request by CheezItMan**
Revised PSE #6 to match the new format.

### CODE REVIEW

File: 06/merging-sorted-lists.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+- [About PSEs](../about-pses/about-pses.md)
+- [Our example PSE with example answers](../about-pses/example-pse.md)
+- Any past PSEs you may have
+
+##### !end-hint
+##### !explanation
+
+Here are some example clarifying questions:
+
+- What about empty lists
```
Comment (kp483) [inline 06/merging-sorted-lists.md:42]: For consistency
```suggestion
- What about empty lists?
```

**Review by kp483 — APPROVED** [review]
Looks good! Just one tiny nitpick

File: 06/merging-sorted-lists.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+- What to do about elements of equal value? for example `[3]` and `[3]`, do both go in result?
+- Can numbers be negative?
```
Comment (beccaelenzil) [inline 06/merging-sorted-lists.md:44]: Minor suggestion to rephrase with complete sentence:

```suggestion
- How does the function handle empty lists?
- How does the function sort elements of equal value? 
    - For example: `list1 = [3]` and `list2 = [3]`. Are both values include in returned list?
- Can numbers in the input lists be negative?
```

File: 06/merging-sorted-lists.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        - append list[j] to result
+        - increment j
+1. while i < length of list 1
+    - append list[i] to result
+    - increment i
+1. while j < length of list 2
+    - append list[j] to result
+    - increment j
+1. return result
+
```
Comment (beccaelenzil) [inline 06/merging-sorted-lists.md:210]: Minor suggestions to be consistent with previous PSEs and update list to `list1` or `list2`

```suggestion
Example steps for an O(n<sup>2</sup>) solution:

1. Check for valid input
1. set `i` to 0 and `j` to 0
1. initialize `result` as an empty list
1. while `i` < length of `list1` and `j` < length of `list2`
    - if `list1[i]` < `list2[j]`
        - append `list1[i]` to `result`
        - increment `i`
    - else
        - append `list2[j]` to `result`
        - increment `j`
1. while `i` < length of `list1`
    - append `list1[i]` to `result`
    - increment `i`
1. while `j `< length of `list2`
    - append `list2[j]` to `result`
    - increment `j`
1. return `result`

```

**Review by beccaelenzil — COMMENTED** [review]
Looks good! Just a few small suggestions. The `replit` is a great addition. Thanks for making that!

**Review by beccaelenzil — APPROVED** [review]

**Pull request merged**